e2e-tests/IIoTPlatform-E2E-Tests/Standalone/A_PublishSingleNodeStandaloneTestTheory.cs

```diff
@@ -66,9 +82,10 @@ public class A_PublishSingleNodeStandaloneTestTheory {
                 })
                 .ToArray();
 
+            cts = new CancellationTokenSource(TimeSpan.FromMinutes(5));
```

Comment:
>TimeSpan.FromMinutes(5) [](start = 46, length = 23)

same as above

Reply:
this one can be the default one, I will change it